index.bs

```diff
@@ -5489,13 +5489,15 @@ dictionary Transformer {
   TransformerStartCallback start;
   TransformerTransformCallback transform;
   TransformerFlushCallback flush;
+  TransformerCancelCallback cancel;
   any readableType;
   any writableType;
 };
 
 callback TransformerStartCallback = any (TransformStreamDefaultController controller);
 callback TransformerFlushCallback = Promise<undefined> (TransformStreamDefaultController controller);
 callback TransformerTransformCallback = Promise<undefined> (any chunk, TransformStreamDefaultController controller);
+callback TransformerCancelCallback = Promise<undefined> (any reason, TransformStreamDefaultController controller);
```

Comment:
Remove the `controller` argument. Both readable and writable sides have already transitioned to a final state by the time we call `cancel()`, so you can no longer use the controller for anything useful.

Also, this aligns with ReadableStream's `cancel()` algorithm and WritableStream's `abort()` algorithm.

Reply:
Done